packages/apps/postgres/README.md

```diff
@@ -66,44 +66,57 @@ See:
+| Name                                    | Description                                                                                                              | Type     | Value   |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | -------- | ------- |
+| `external`                              | Enable external access from outside the cluster                                                                          | `bool`   | `false` |
+| `size`                                  | Persistent Volume size                                                                                                   | `string` | `10Gi`  |
+| `replicas`                              | Number of Postgres replicas                                                                                              | `int`    | `2`     |
+| `storageClass`                          | StorageClass used to store the data                                                                                      | `string` | ``      |
+| `postgresql`                            | PostgreSQL server configuration                                                                                          | `object` | `null`  |
+| `postgresql.parameters`                 | PostgreSQL server parameters                                                                                             | `object` |         |
+| `postgresql.parameters.max_connections` | Determines the maximum number of concurrent connections to the database server. The default is typically 100 connections | `int`    |         |
+| `quorum`                                | Quorum configuration for synchronous replication                                                                         | `object` | `null`  |
+| `quorum.minSyncReplicas`                | Minimum number of synchronous replicas that must acknowledge a transaction before it is considered committed.            | `int`    |         |
+| `quorum.maxSyncReplicas`                | Maximum number of synchronous replicas that can acknowledge a transaction (must be lower than the number of instances).  | `int`    |         |
+
+### Configuration parameters
+
+| Name                             | Description                                 | Type                  | Value |
+| -------------------------------- | ------------------------------------------- | --------------------- | ----- |
+| `users`                          | Users configuration                         | `map[string]user`     | `{}`  |
+| `users[name].password`           | Password for the user                       | `*string`             |       |
+| `users[name].replication`        | Whether the user has replication privileges | `*bool`               |       |
```

Comment:
It's not quite clear what `*bool` is. Optional? Anyway, a boolean should always have a default value, but this line has nothing in the fourth column.